sonarqube/tests/conftest.py

```diff
@@ -8,42 +8,35 @@
 import pytest
 
 from datadog_checks.dev import TempDir, docker_run
-from datadog_checks.dev.conditions import CheckDockerLogs, CheckEndpoints
+from datadog_checks.dev.conditions import CheckDockerLogs
 from datadog_checks.sonarqube import SonarqubeCheck
 
 from . import common
 
 
 @pytest.fixture(scope='session')
 def dd_environment():
-    compose_file = os.path.join(common.HERE, 'docker', 'docker-compose.yaml')
+    compose_file = os.path.join(common.HERE, 'docker', common.COMPOSE_FILE)
 
     # The scanner creates artifacts within the project such as `.scannerwork/`
     with TempDir('sonarqube-project') as temp_dir:
         project_dir = os.path.join(temp_dir, 'project')
         if not os.path.isdir(project_dir):
             shutil.copytree(os.path.join(common.HERE, 'docker', 'project'), project_dir)
-
         with docker_run(
             compose_file,
-            service_name='sonarqube',
             env_vars={'PROJECT_DIR': project_dir},
             conditions=[
                 CheckDockerLogs('sonarqube', ['SonarQube is up'], attempts=100, wait=3),
-                CheckEndpoints([common.WEB_INSTANCE['web_endpoint']]),
+                CheckDockerLogs('sonar-scanner', ['ANALYSIS SUCCESSFUL'], attempts=100, wait=3),
+                CheckDockerLogs(
+                    'sonarqube', ['Executed task | project=org.sonarqube:sonarqube-scanner'], attempts=100, wait=3
+                ),
```

Comment:
Could we add the endpoint again just in case?

Reply:
The right way to check it in docker-compose is `healthcheck:` in `sonarqube` service

Reply:
Agreed. Do we have something on the python side to wait for the container to be healthy?

Reply:
The concept and conditions of a container to be `healthy` may vary from one to another. For example, one container can be healthy when after starting it is listening on port 8080, however another one can be `healthy` when it has started and read all the records from an init file (or a database).

Reply:
right but I wanted to know if we have something on our python side to check that the container healthy check is set to healthy for this reason.

If you think those conditions are enough, let's keep it as it is then

Reply:
`docker` offers an API through which we could query it from our python code, but I'm afraid we don't have it implemented (at least I don't know)

Reply:
ok thanks! might be something to consider in the future